src/collections/lookup-map.ts

```diff
@@ -1,37 +1,40 @@
 import * as near from '../api'
+import { Bytes } from '../utils';
 
 export class LookupMap {
-    constructor(keyPrefix) {
+    readonly keyPrefix: Bytes;
+
+    constructor(keyPrefix: Bytes) {
         this.keyPrefix = keyPrefix
     }
 
-    containsKey(key) {
+    containsKey(key: Bytes): boolean {
```

Comment:
What is the UX here? Can user just pass a string?

Reply:
Opened https://github.com/near/near-sdk-js/issues/117 to track this